control-services/cli/vDocker.py

```diff
@@ -29,6 +29,15 @@ def compose_stop(self, containers):
 
         subprocess.check_output(cmd, universal_newlines=True)
 
+    def compose_rm(self, containers):
+        containers = self.__makeStrList(containers)
+
+        cmd = self.composePreface + [
+                'rm', '-sf',
```

Comment:
What is -sf? Didn't find it in the [docker docs](https://docs.docker.com/engine/reference/commandline/rm/#options)

Reply:
`self.composePreface` adds `docker-compose`:

```
docker-compose rm
Usage: rm [options] [SERVICE...]

Options:
    -f, --force   Don't ask to confirm removal
    -s, --stop    Stop the containers, if required, before removing
    -v            Remove any anonymous volumes attached to containers
    -a, --all     Deprecated - no effect.
```